inference-engine/.clang-format

```diff
@@ -0,0 +1,54 @@
+BasedOnStyle: LLVM
```

Comment:
As I know at the current moment we use the google style (it looks like this config: https://github.com/openvinotoolkit/openvino/blob/master/thirdparty/itt_collector/.clang-format, I used cpplint rules to create this file).

Am I right that that you have used nGraph config as a baseline?

@ilya-lavrenov What do you think should we align samples with nGraph or IE code style?

In additional, if you enable clang-format only for samples, please add this file only to samples folder, because in other case developers, who have integrated clang-format to editors, will have problems, because editor will automatically apply this code style to all changed files.

Reply:
I take this format file from https://github.com/openvinotoolkit/openvino/tree/master/ngraph/ folder. Ok I will move it to the sample folder.

Reply:
I took google style.